README.md

```diff
@@ -56,6 +56,18 @@ Example:
         name: Bar User
         uid: 1002
 
+## Creating users for SSH tunnel
+Sometimes we need to create user which will be used for ssh tunnels only. In this case add option 'tunnel_only' to user arguments, like this:
+
+    users:
+      - username: foo-tun
+        name: Foo Tunnel
+        groups: []
+        uid: 1234
+        tunnel_only: True
```

Comment:
I think it might make more sense to have an optional generic `ssh_key_options` parameter that can take a string of ssh key options. It simplifies the logic a bit and removes the need to hard code any tunnel options into the role.

Something like this: `ssh_key_options: "{{users_default_ssh_tunnel_options}}"`.